src/storage/engine/mod.rs

```diff
@@ -31,22 +31,33 @@ const SEEK_BOUND: usize = 30;
 #[derive(Debug)]
 pub enum Modify {
     Delete(Key),
```

Comment:
@BusyJay Just reconsidered about it. Currently we use `Modify::Delete`/`Modify::Put` to construct a Modify in many places, keep the old version will make them more simple.